mod-oai-pmh/src/test/resources/common/setup-users.feature

```diff
@@ -13,6 +13,7 @@ Feature: setup test users
     {
       "id": #(testUser.id),
       "username": '#(testUser.name)',
+      "barcode": #(testUser.barcode),
```

Comment:
Why did we have to change anything in mod-oai-pmh?

Reply:
I added this field, so that we can use testUser as a proxy user.

Reply:
I mean, why do we depend on mod-oai-pmh at all?